public/app/features/alerting/unified/components/silences/SilencesTable.tsx

```diff
@@ -29,12 +32,36 @@ export interface SilenceTableItem extends Silence {
 type SilenceTableColumnProps = DynamicTableColumnProps<SilenceTableItem>;
 type SilenceTableItemProps = DynamicTableItemProps<SilenceTableItem>;
 interface Props {
-  silences: Silence[];
-  alertManagerAlerts: AlertmanagerAlert[];
   alertManagerSourceName: string;
 }
 
-const SilencesTable = ({ silences, alertManagerAlerts, alertManagerSourceName }: Props) => {
+const SilencesTable = ({ alertManagerSourceName }: Props) => {
+  const { data: alertManagerAlerts, isLoading: amAlertsIsLoading } =
+    alertmanagerApi.endpoints.getAlertmanagerAlerts.useQuery(
+      { amSourceName: alertManagerSourceName, filter: { silenced: true, active: true, inhibited: true } },
+      { pollingInterval: SILENCES_POLL_INTERVAL_MS }
+    );
+
+  const {
+    data: silences = [],
+    isLoading,
+    error,
+  } = alertSilencesApi.endpoints.getSilences.useQuery(
+    { datasourceUid: getDatasourceAPIUid(alertManagerSourceName) },
+    {
+      pollingInterval: SILENCES_POLL_INTERVAL_MS,
+    }
+  );
+
+  const { currentData: amFeatures } = featureDiscoveryApi.useDiscoverAmFeaturesQuery(
+    { amSourceName: alertManagerSourceName ?? '' },
+    { skip: !alertManagerSourceName }
+  );
+
+  const mimirLazyInitError =
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Any suggestions/patterns we're using elsewhere to handle errors? BackendSrv (I think) is giving it back to us always as `unknown`

Reply:
(now using `stringifyErrorLike` as per Gilles' suggestion elsewhere)